examples/1.0.0/bnpl-arazzo.yaml

```diff
@@ -0,0 +1,256 @@
+arazzo: 1.0.0
+info:
+  title: BNPL Workflow description
+  version: 1.0.0
+sourceDescriptions:
+  - name: BnplApi
+    url: https://raw.githubusercontent.com/OAI/Arazzo-Specification/main/examples/1.0.0/bnpl-openapi.yaml
+    type: openapi
+workflows:
+- workflowId: ApplyForLoanAtCheckout
+  summary: Apply for a loan at checkout using a BNPL platform
+  description: Describes the steps to secure a loan at checkout from a BNPL platform. It is a multistep process that requires multiple API calls across several API providers to be completed successfully.
+  inputs:
+      type: object
+      required:
+      - customer
+      - products
+      properties:
+        customer:
+          description: |
+            Customer can either be the customer details, which will be used for enrollment, or a link to an existing customer resource as the customer already uses the BNPL platform
+          oneOf:
+          - type: object
+            required:
+            - firstName
+            - lastName
+            - dateOfBirth
+            - postalCode
+            properties:
+              firstName:
+                description: First name of customer
+                type: string
+                minLength: 1
+                maxLength: 70
+              lastName:
+                description: Last name of customer
+                type: string
+                minLength: 1
+                maxLength: 70
+              dateOfBirth:
+                description: Customer date of birth
+                type: string
+                format: date-time
+              postalCode:
+                description: Zip code or postal code of customer
+                type: string
+                minLength: 1
+                maxLength: 70
+              additionalProperties: false
```

Comment:
```suggestion
unevaluatedProperties: false
```

Reply:
for consistency with using JSON Schema 2020-12

Reply:
But there are no other applicators, so they have the same result, except `unevaluated` is substantially more complex and computationally expensive (at least in my implementation, can't speak for any other)